code/game/objects/items/_inventory.dm

```diff
@@ -63,12 +63,12 @@
 
 /obj/item/proc/is_worn()
 	if (ismob(loc))
-		return !(equip_slot in list(slot_none, slot_l_hand, slot_r_hand))
+		return !(equip_slot in list(slot_none, slot_l_hand, slot_r_hand,slot_robot_equip_1,slot_robot_equip_2,slot_robot_equip_3))
 
 
 /obj/item/proc/is_held()
 	if (ismob(loc))
-		return equip_slot in list(slot_l_hand, slot_r_hand)
+		return equip_slot in list(slot_l_hand, slot_r_hand,slot_robot_equip_1,slot_robot_equip_2,slot_robot_equip_3)
```

Comment:
Same here, since there is now `unworn_slots` it would make sense to have `unheld_slots`. Though I question the naming scheme and whether having `worn_slots` and `held_slots` wouldn't be better.

Reply:
pocket slots are neither worn nor held. They would not be in held slots